kpi/tests/api/v2/test_api_attachments_delete_viewset.py

```diff
@@ -296,14 +386,14 @@ def test_bulk_delete_attachments_empty_uid_list(self):
         assert response.status_code == status.HTTP_400_BAD_REQUEST
         assert response.data == {
             'submission_root_uuids': [
-                ErrorDetail(
-                    string='Submission root UUIDs list cannot be empty', code='invalid'
-                )
+                ErrorDetail(string='List cannot be empty', code='invalid')
             ]
         }
         assert AttachmentTrash.objects.count() == initial_trash_count
-        for uid in self.attachment_uids:
-            assert not AttachmentTrash.objects.filter(uid=uid).exists()
```

Comment:
This assertion was wrong. We were comparing Attachment uids with AttachmentTrash uids